src/elements/link/index.js

```diff
@@ -0,0 +1 @@
+export default from './link'
```

Comment:
I do not like this convention. Do you see any other better option to handle component modules?

Reply:
Not at the moment. :(